src/elements/flip-card/flip-card-details/flip-card-details.ts

```diff
@@ -0,0 +1,68 @@
+import type { CSSResultGroup, PropertyValues, TemplateResult } from 'lit';
+import { html, LitElement } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+import { IS_FOCUSABLE_QUERY } from '../../core/a11y.js';
+import { hostAttributes } from '../../core/decorators.js';
+import { AgnosticMutationObserver } from '../../core/observers.js';
+
+import style from './flip-card-details.scss?lit&inline';
+
+/**
+ * Combined with a `sbb-flip-card`, it displays its content when the card is flipped.
+ *
+ * @slot - Use the unnamed slot to provide any kind of content.
+ */
+@customElement('sbb-flip-card-details')
+@hostAttributes({
+  slot: 'details',
+})
+export class SbbFlipCardDetailsElement extends LitElement {
+  public static override styles: CSSResultGroup = style;
+
+  private _flipCardMutationObserver = new AgnosticMutationObserver(() =>
+    this._checkForSlottedActions(),
+  );
+
+  private _checkForSlottedActions(): void {
+    const cardFocusableAttributeName = 'data-card-focusable';
+
+    Array.from(this.querySelectorAll?.(IS_FOCUSABLE_QUERY) ?? [])
+      .filter((el) => !el.hasAttribute(cardFocusableAttributeName))
+      .forEach((el: Element) => el.setAttribute(cardFocusableAttributeName, ''));
+  }
+
+  public override connectedCallback(): void {
+    super.connectedCallback();
+    this._checkForSlottedActions();
+    this._flipCardMutationObserver.observe(this, {
+      childList: true,
+      subtree: true,
+    });
+  }
+
+  protected override firstUpdated(changedProperties: PropertyValues): void {
+    super.firstUpdated(changedProperties);
+    this._checkForSlottedActions();
+  }
+
+  public override disconnectedCallback(): void {
+    super.disconnectedCallback();
+    this._flipCardMutationObserver.disconnect();
+  }
+
+  protected override render(): TemplateResult {
+    return html`
+      <div class="sbb-flip-card-details--wrapper">
```

Comment:
Nitpick; Is the --wrapper part necessary? Usually the inner class just corresponds to the element name?